### DISCUSSION TITLE
[adonisJs5]   Error validating Json Object

### DISCUSSION THREAD

**Discussion by andre-ab** [adonisjs/core#4348] [General]
I'm new to AdonisJs, if it's not on the forum, where can I post?

I'm trying to validate this json:
![image](https://github.com/adonisjs/core/assets/10952239/f24acccb-82e2-4bc1-9cf8-4eb34258020e)

but it is returning this error:
![image](https://github.com/adonisjs/core/assets/10952239/2a54830c-e551-490b-8f4a-58a3d60f33a4)

validation cod: 
![image](https://github.com/adonisjs/core/assets/10952239/e5ce724e-eee7-44bc-b3cd-2d9fc95c8003)

**Comment by RomainLanz** [comment]
Hey @andre-ab! 👋🏻 

You are wrongly using the validator. I would recommend [to look at the documentation](https://docs.adonisjs.com/guides/validator/introduction#standalone-validator-usage) again.

For example, you are not using `schema.create` to create the schema [when using `request.validate`](https://docs.adonisjs.com/guides/validator/introduction#validating-http-requests).

**Reply by andre-ab** [reply]
thank you for the informations
